src/components/newNoteInput/NewNoteInput.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState, useEffect, useRef } from "react";
+import "./newNoteInput.css";
+import ReactQuill from "react-quill";
+import "react-quill/dist/quill.snow.css";
+import { quillModules as modules } from "../../utils/quillModules";
+import { PinIcon } from "../../assets";
+import { addToSavedNotes } from "../../services/firebaseServices";
+
+const NewNoteInput = ({ user }) => {
+  const [value, setValue] = useState("");
+  const editorRef = useRef();
+  const [showEditor, setShowEditor] = useState(false);
+
+  const initialNoteState = {
+    title: "",
+    content: "",
+    noteColor: "FFFFFF",
+    isPinned: false,
+    tags: [],
+  };
+
+  const [note, setNote] = useState(initialNoteState);
+
+  const updateNoteContent = () => {
+    setNote(() => ({ ...note, content: value }));
+  };
+
+  useEffect(() => {
+    updateNoteContent();
+  }, [value]);
+
+  const addNoteHandler = () => {
+    const { title, content } = note;
+    if (title !== "" || content !== "") {
+      addToSavedNotes(user, note);
+
+      setNote(initialNoteState);
+      setValue("");
+      setShowEditor(false);
+    } else {
+      alert("Please add atleast title or note");
+    }
```

Comment:
Suggestion : Try using a toast instead of alert

Reply:
Yess. I know... Will implement it in next PR